src/Privacy.elm

```diff
@@ -0,0 +1,90 @@
+module Privacy exposing (text)
+
+
+text =
+    """
+
+# Privacy Policy
+#### Version 1.0 – June 2021
+
+We’re committed to protecting and respecting your privacy. If you have any questions about your personal information please chat with us by emailing us at [hello@meetdown.app](mailto:hello@meetdown.com).
+
+
+### 👀 The information we hold about you
+
+#### - Cookie information
+
+We use a single persistent secured httpOnly session cookie to recognise your browser and keep you logged in.
+
+Other cookies may be introduced in the future, and if so our Privacy policy will be updated at that time.
+
+
+#### - Information submitted through our service or website
+
+- For example, when you sign up to the service and provide details such as your name and email
+
+There may be times when you give us ‘sensitive’ information, which includes things like your racial origin, political opinions, religious beliefs, trade union membership details or biometric data. We’ll only use this information in strict accordance with the law.
+
+
+### 🔍 How we use your information
+
+To provide our services, we use it to:
+
+- Help us manage your account
+- Send you reminders for events you've joined
+
+To meet our legal obligations, we use it to:
+
+- Prevent illegal activities like piracy and fraud
+
+With your permission, we use it to:
+
+- Market and communicate our products and services where we think these will be of interest to you by email. You can always unsubscribe from receiving these if you want to by email.
```

Comment:
Obvs we don't have this feature/mechanism, but it feels sensible in case we need to contact everyone in future by email to be like "Hey, here's what's available now, also you can now manage your subscription options to this.".